pybind11/setup_helpers.py

```diff
@@ -59,8 +59,7 @@
 import distutils.errors
 import distutils.ccompiler
 
-
-WIN = sys.platform.startswith("win32") and sysconfig.get_platform() != "mingw"
+WIN = sys.platform.startswith("win32") and "mingw" not in sysconfig.get_platform()
```

Comment:
Mind pulling out _just_ this change in a PR? I'd like to make a patch release soon and it looks like the other parts might not be solved in time, but this is a clear fix for something we list in the release nodes for 2.7.0.

Reply:
Nevermind. Looks like this needs to be fixed first.